pytorch_lightning/loggers/tensorboard.py

```diff
@@ -115,13 +115,8 @@ def log_hyperparams(self, params: Union[Dict[str, Any], Namespace],
             )
         else:
             from torch.utils.tensorboard.summary import hparams
```

Comment:
```suggestion
```

Isn't needed anymore right?

Reply:
It crashes if sanitized_params or metrics is None. Tensorboard doc says it is okay, but code checks for dict or raises an error.

[Doc](https://pytorch.org/docs/stable/tensorboard.html)
[Code](https://pytorch.org/docs/stable/_modules/torch/utils/tensorboard/writer.html#SummaryWriter.add_hparams)